### DISCUSSION TITLE
Haunted By Ghost Notifications...

### DISCUSSION THREAD

**Discussion by Ibuprophen** [community/community#24471] [Apps, API and Webhooks]
<p>Hello There! :slight_smile:</p>
<p>I have a Ghost Notification that never goes away (via multiple PC and Android Browser’s) and I’ve done everything I can think of to get rid of it.</p>
<p>I’ve even removed every single repo that I had marked as following and it still remains.</p>
<p>The following 2 images will reflect what I mean…</p>
<p><a href="https://github.com/ibuprophen1/Odds-N-Ends/raw/master/TEMP/Github-Ghost-Notification-1.png" rel="nofollow noopener">https://github.com/ibuprophen1/Odds-N-Ends/raw/master/TEMP/Github-Ghost-Notification-1.png</a></p>
<p><a href="https://github.com/ibuprophen1/Odds-N-Ends/raw/master/TEMP/Github-Ghost-Notification-2.png" rel="nofollow noopener">https://github.com/ibuprophen1/Odds-N-Ends/raw/master/TEMP/Github-Ghost-Notification-2.png</a></p>
<p>It’s been like this for approximately 6 months +/- as well.</p>
<p>Please help me! :::frowning:</p>
<p>~Ibuprophen</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/ibuprophen1">@ibuprophen1</a>,</p>
<p>Thanks for reaching out! That’s very strange I have not seen that behavior elsewhere, are you still having this issue?</p>

**Comment by Ibuprophen** [comment]
<p>Yes I do…</p>
<p>I tried the email approach before this forum submission and received a response asking this similar question. I responded to it and still haven’t heard anything back.</p>
<p>This is a really odd issue that started without any rhyme to reason.</p>
<p>~Ibuprophen/Ibuprophen1</p>

**Comment by Ibuprophen** [comment, marked as answer]
<p>Just an FYI…</p>
<p>This issue was just taken care of by Steve Guntrip (GitHub Support Staff) via email.</p>
<p>I’m not completely sure “what, where why &amp; how” but, I’m just happy it’s all set now.</p>
<p>Thank you very much for your time and understanding with this matter and has now been resolved.</p>
<p>In the future, I’m going to stick with the email support since there seems to be very little interest in obtaining it from this Community Forum.</p>
<p>~Ibuprophen1</p>

**Comment by MathiasWP** [comment]
This happened to me as well, and this solution helped me fix it:

1. Go to https://github.com/notifications?query=is%3Aunread
2. Run this in the browser's debug console:

	```js
	document.querySelector('.js-notifications-mark-all-actions').removeAttribute('hidden');
	document.querySelector('.js-notifications-mark-all-actions form[action="/notifications/beta/archive"] button').removeAttribute('disabled');
	```
3. Press the previously hidden "Done" button.

Credit: https://stackoverflow.com/a/79774035

**Reply by addy-zero** [reply]
This did the trick for me, but do we know why this happened?
